environments/staging/deploys.yml

```diff
@@ -933,3 +933,51 @@ bare-template.aragonpm.eth:
       ipfsHash: QmbB5XPhn4wgF3zLYDBzbbwcx8yaVemwDZZzX6GJz8HMe4
       contractAddress: '0x6761aE15235f137daFBdA6C79af627697c61BDb5'
       commitHash: 72d9d0ccfd2c0a88eb2c4c664150471887d07b37
+aragon-fundraising.aragonpm.eth:
```

Comment:
Why only refer to `2.0.0` and not also `1.0.1` and `2.0.1` ?

Reply:
I accidentally messed up the 1.0.0 and 2.0.0 deployments (bad contract addresses or content urls); now it's on 3.0.0.

Didn't want to replicate the _entire_ state of the repos in rinkeby (since staging shouldn't matter as much, and we don't even sync mainnet and rinkeby deployments for other repos), so didn't bother with the interim versions on any packages.